src/include/autoinstall/ask.rb

```diff
@@ -136,7 +176,7 @@ def askDialog
           title = Ops.get_string(ask, "title", "")
           back_label = Ops.get_string(ask, "back_label", back_label)
           ok_label = Ops.get_string(ask, "ok_label", ok_label)
-          timeout = Ops.get_integer(ask, "timeout", 0)
+          timeout = ask["timeout"] if ask.key?("timeout")
```

Comment:
The surrounding code is absolutely awful (everything is inlined) so it is very hard to review whether this does not break some other use case. I would rather revert this part of the fix.